Dockerfile

```diff
@@ -62,4 +62,6 @@ COPY --from=server-build /app/server/build ./build
 
 EXPOSE 80
 
-ENTRYPOINT ["./entrypoint.sh"]
+HEALTHCHECK NONE
```

Comment:
`HEALTHCHECK NONE` is redundant in this case. Either implement one (which would be great), or delete this line

Reply:
I had Hadolint linter activated.
https://github.com/hadolint/hadolint/wiki/DL3057

But yes, I'll add something.

Reply:
Added